packages/compass-indexes/src/modules/regular-indexes.spec.ts

```diff
@@ -54,35 +53,35 @@ describe('regular-indexes module', function () {
     }
   });
 
-  it('#sortIndexes action - it sorts indexes as defined', async function () {
+  it('#sortIndexes action - it sorts indexes as defined', function () {
     store.dispatch(setIndexes(defaultSortedDesc as any));
 
     {
-      await store.dispatch(sortIndexes('Name and Definition', 'asc'));
+      store.dispatch(sortIndexes('Name and Definition', 'asc') as any);
```

Comment:
You need to assert here because `Store<RootState>` above is not a correct type for the store (it doesn't take thunk into account), if you change it to `ReturnType<typeof setupStore>` the need for this should go away

Reply:
oh that's awesome :) i spent time figuring this out and ended up with `any`.

Reply:
One cool (but totally not obvious thing) about redux middleware (not only thunk one) types is that it can actually change the type of store and dispatch methods, very handy technique